apps-rendering/cdk/lib/__snapshots__/mobile-apps-rendering.test.ts.snap

```diff
@@ -988,15 +1034,14 @@ exports[`The MobileAppsRendering stack matches the snapshot 1`] = `
               "Fn::Join": [
                 "",
                 [
-                  "#!/bin/bash -ev
+                  "#!/bin/bash
+#!/bin/bash -ev
```

Comment:
@akash1810 @jacobwinch This is interesting it looks like CDK now automatically adds its own shebang. iiuc the second shebang added by our own userdata will be ignored as a comment which means we lose the -ev options i.e. early exit and verbose mode.

Reply:
This part of the AWS CDK codebase hasn't changed in a while - https://github.com/aws/aws-cdk/blame/e7a6e14d7c3ddfbff8b1fd3f583abeefeed1258a/packages/aws-cdk-lib/aws-ec2/lib/user-data.ts#L172-L175, so I'm a little confused how this diff has occurred 🤔.

Reply:
Would that be because we've only just switched to `UserData.forLinux()` in this PR?

Reply:
> Would that be because we've only just switched to `UserData.forLinux()` in this PR?

🤦🏽 Ah yes! Sorry, I was only looking at the snapshot file.

Reply:
Np. Thanks for the guidance